template/build/vue-loader.conf.js

```diff
@@ -1,19 +1,9 @@
 'use strict'
```

Comment:
@gkatsanos, they seem to work well on my machine... The CSS file in `dist` is minified and when I click a selector, Chrome opens the `.vue` file and highlights the proper line.

Can you point me to something specific so I can verify?

Reply:
@afontcu, I think it should stay, but I'm not really sure.

Vue is very new for me, so I don't know how every setting affects the build or the app. I only made changes I felt certain were equivalent to the previous behavior or completely redundant, mainly by following Vue's [migration guide](https://vue-loader.vuejs.org/migrating.html), reading docs, and trial and error.

For example, I got rid of the `loaders` property because the new vue-loader explicitly deprecates it in favor of the the webpack loaders configuration. And I renamed the `transformToRequire` property to `transformAssetUrls`, based on the guide.

My hunch is that it should probably stay for the sake of clarity, even though the usage has changed a bit.

Having said that, I haven't checked if the `cacheBusting` property is used anywhere, and I'm not sure if we should add other properties that are relevant for the new vue-loader.